### PR TITLE
Update Bootstrap 3.3.6 and Bootswatch 3.3.6+1

### LINKED ISSUE #108: Slow loading speed
I am not sure whether this is a sphinx-bootstrap-theme issue or not.
I am using sphinx-bootstrap-theme for my online help.

It slows down page loading on the pages if I specify 'bootswatch_theme': "flatly" or other theme options. My help is fine if I don't specify that option.

My console said "Fail to load resource :  net::ERR_FILE_NOT_FOUND ... file://fonts.googleapis.com/css?family=Lato:400,700,900,400italic

I am using latest version of Google Chrome(36.0.1985.143), and the help is built with Sphinx (sphinx-build) 1.2 

Any ideas or help?

Thank you.

### PR DESCRIPTION
**Pull request by ppyv**
Introducing new version of Bootstrap and Bootswatch. I referred 4030ff4 and this also fixes #108.

### CODE REVIEW

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
- * Bootstrap v3.3.4 (http://getbootstrap.com)
+ * Bootstrap v3.3.6 (http://getbootstrap.com)
  * Copyright 2011-2015 Twitter, Inc.
- * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ * Licensed under the MIT license
  */
 
 if (typeof (window.$jqTheme || window.jQuery) === 'undefined') {
-  throw new Error('Bootstrap\'s JavaScript requires jQuery')
+  throw new Error('Bootstrap\'s JavaScript requires (window.$jqTheme || window.jQuery)')
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 }
 
 +function ($) {
   'use strict';
-  var version = $.fn.jquery.split(' ')[0].split('.')
-  if ((version[0] < 2 && version[1] < 9) || (version[0] == 1 && version[1] == 9 && version[2] < 1)) {
-    throw new Error('Bootstrap\'s JavaScript requires jQuery version 1.9.1 or higher')
+  var version = $.fn.(window.$jqTheme || window.jQuery).split(' ')[0].split('.')
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+  if ((version[0] < 2 && version[1] < 9) || (version[0] == 1 && version[1] == 9 && version[2] < 1) || (version[0] > 2)) {
+    throw new Error('Bootstrap\'s JavaScript requires (window.$jqTheme || window.jQuery) version 1.9.1 or higher, but lower than version 3')
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 4]
```diff
@@ -1241,9 +1247,9 @@ if (typeof (window.$jqTheme || window.jQuery) === 'undefined') {
 }((window.$jqTheme || window.jQuery));
 
 /* ========================================================================
- * Bootstrap: tooltip.js v3.3.4
+ * Bootstrap: tooltip.js v3.3.6
  * http://getbootstrap.com/javascript/#tooltip
- * Inspired by the original jQuery.tipsy by Jason Frame
+ * Inspired by the original (window.$jqTheme || window.jQuery).tipsy by Jason Frame
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 5]
```diff
@@ -2015,10 +2059,12 @@ if (typeof (window.$jqTheme || window.jQuery) === 'undefined') {
   // ====================
 
   var Tab = function (element) {
+    // jscs:disable requireDollarBefore(window.$jqTheme || window.jQuery)Assignment
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

File: sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js [hunk 6]
```diff
@@ -2015,10 +2059,12 @@ if (typeof (window.$jqTheme || window.jQuery) === 'undefined') {
...
     this.element = $(element)
+    // jscs:enable requireDollarBefore(window.$jqTheme || window.jQuery)Assignment
```
Comment (ryan-roemer) [inline sphinx_bootstrap_theme/bootstrap/static/bootstrap-3.3.6/js/bootstrap.js]: Bad search and replace here.

**Comment by ryan-roemer** [comment]
@ppyv -- Thanks for the PR! I've identified a few instances where search/replacing on `jQuery` needs a little more manual effort, but after those are fixed I'll pull it down and try it out!

**Comment by ppyv** [comment]
@ryan-roemer Thanks for your comment and I've made a commit for replacing those. Could you check it?

**Comment by ppyv** [comment]
ping, any comments?

**Comment by ryan-roemer** [comment]
Testing out now. There's still some bad search and replace. In the min version we've got:

```js
a.fn.jQuery.split(" ")[0].split(".")
```

when it needs to be:

```js
a.fn.jquery.split(" ")[0].split(".")
```

I'm going to try and see if I can fix things manually.

**Comment by ryan-roemer** [comment]
Released in https://pypi.python.org/pypi/sphinx-bootstrap-theme/0.4.9 

Thanks for all the work @ppyv !

**Pull request merged**